skyportal/handlers/api/galaxy.py

```diff
@@ -104,11 +105,37 @@ def get(self, catalog_name=None):
           tags:
             - galaxies
           parameters:
-            - in: catalog_query
-              name: name
+            - in: query
+              name: catalog_name
               schema:
                 type: string
               description: Filter by catalog name (exact match)
+            - in: query
+              name: localizationDateobs
+              schema:
+                type: string
+              description: |
+                Event time in isot format
+            - in: query
+              name: localizationName
+              schema:
+                type: string
+              description: |
+                Name of localization / skymap to use
```

Comment:
Guidance for the user on where to find these, or what they can be?

Reply:
Added.